model/host/static_hosts.go

```diff
@@ -20,14 +20,9 @@ import (
 // If the distro is the empty string ("") then this operation affects
 // all distros.
 func MarkInactiveStaticHosts(activeStaticHosts []string, distroID string) error {
-	if len(activeStaticHosts) == 0 {
```

Comment:
in scheduler/wrapper.go:200 we still check this and skip the update

Reply:
Fixed.